shell/platform/darwin/macos/framework/Source/FlutterKeyboardManagerUnittests.mm

```diff
@@ -356,6 +364,40 @@ - (bool)textInputPlugin {
   return true;
 }
 
+- (bool)forwardKeyEventsToSystemWhenComposing {
+  KeyboardTester* tester = [[KeyboardTester alloc] init];
+
+  tester.isComposing = YES;
+  // Send a down event with composing == YES.
+  [tester.manager handleEvent:keyDownEvent(0x50)];
+
+  NSMutableArray<FlutterAsyncKeyCallback>* channelCallbacks =
+      [NSMutableArray<FlutterAsyncKeyCallback> array];
+  NSMutableArray<FlutterAsyncKeyCallback>* embedderCallbacks =
+      [NSMutableArray<FlutterAsyncKeyCallback> array];
+  [tester recordEmbedderCallsTo:embedderCallbacks];
+  [tester recordChannelCallsTo:channelCallbacks];
+  // TextInputPlugin does not claim the event.
+  [tester respondTextInputWith:NO];
```

Comment:
This method is used to set the responses from the mocked TextInputPlugin, not to check. (Maybe I should add a method to check its value...)

Reply:
Yeah I didn't mean to use this as a "verification" statement. The verification step is at line 385 & 386 where we make sure the event doesn't go to the primary responder(s) even with text input not claiming the key event for itself.

Reply:
Yeah so this line does little and should be removed. And still we haven't ensured that `onTextInputKeyEvent` is called.